front/src/modules/views/components/UpdateViewButtonGroup.tsx

```diff
@@ -38,53 +39,49 @@ export const UpdateViewButtonGroup = ({
 
   const canPersistView = canPersistFilters || canPersistSorts;
 
-  const handleArrowDownButtonClick = useCallback(() => {
-    setIsDropdownOpen((previousIsOpen) => !previousIsOpen);
-  }, []);
+  const handleArrowDownButtonClick = useCallback(() => {}, []);
 
   const handleCreateViewButtonClick = useCallback(() => {
     setViewEditMode('create');
     onViewEditModeChange?.();
-    setIsDropdownOpen(false);
   }, [setViewEditMode, onViewEditModeChange]);
 
-  const handleDropdownClose = useCallback(() => {
-    setIsDropdownOpen(false);
-  }, []);
-
   const handleViewSubmit = async () => {
     await updateCurrentView?.();
   };
 
-  useScopedHotkeys(
-    [Key.Enter, Key.Escape],
-    handleDropdownClose,
-    hotkeyScope,
-    [],
-  );
-
-  if (!canPersistView) return null;
+  if (!canPersistView) {
+    return <></>;
+  }
 
   return (
-    <StyledContainer>
-      <ButtonGroup size="small" accent="blue">
-        <Button title="Update view" onClick={handleViewSubmit} />
-        <Button
-          size="small"
-          Icon={IconChevronDown}
-          onClick={handleArrowDownButtonClick}
-        />
-      </ButtonGroup>
-
-      {isDropdownOpen && (
-        <DropdownMenuItemsContainer>
-          <MenuItem
-            onClick={handleCreateViewButtonClick}
-            LeftIcon={IconPlus}
-            text="Create view"
-          />
-        </DropdownMenuItemsContainer>
-      )}
-    </StyledContainer>
+    <DropdownScope dropdownScopeId={UpdateViewDropdownId}>
```

Comment:
this is the new way of doing it. We have a Dropdown component that we use and not compose our own Dropdown manually